test/fprime_gds/common/loaders/resources/RefTopologyDictionary.json

```diff
@@ -1,6 +1,6 @@
 {
   "metadata" : {
-    "deploymentName" : "Ref",
+    "deploymentName" : "Ref.Ref",
```

Comment:
Shouldn't the deployment be just `Ref`?  Otherwise this is the *topology* name...

Reply:
The `deploymentName` field contains the fully-qualified topology name. In this case, the name of the topology is `Ref` and it exists in the `Ref` module.

Reply:
This is a slight difference between how FPP views deployments and how F Prime/CMake views them.

That is FPP refers to the topology "as the deployment" where as F Prime refers to the build module because it does not know of model information in the build definitions.  This results in `Ref.Ref` vs `Ref`.

This isn't a problem, but I do appreciate the clarification.  It helps!